### PR TITLE
Matched last function in Ovl_en_encount2

### PR DESCRIPTION
**Pull request by hensldm**
Before opening this PR, ensure the following:
- `./format.sh` was run to apply standard formatting.
- `make` successfully builds a matching ROM.
- No new compiler warnings were introduced during the build process.
    - Can be verified locally by running `tools/warnings_count/check_new_warnings.sh`
- New variables & functions should follow standard naming conventions.
- Comments and variables have correct spelling.
---

### CODE REVIEW

File: include/variables.h [hunk 1]
```diff
@@ -4067,7 +4067,8 @@ extern Gfx D_04075B30[];
 extern Gfx D_04076BC0[];
 extern Gfx D_04077480[];
 extern UNK_TYPE D_04079B10;
-extern UNK_TYPE D_0407AB10;
+extern Gfx* D_0407AB10[];
+extern Gfx* D_0407AB58[];
```
Comment (EllipticEllipsis) [inline include/variables.h:4071]:
```suggestion
extern Gfx D_0407AB10[];
extern Gfx D_0407AB58[];
```
Displaylists are arrays, and I doubt these are arrays of pointers to displaylists.

**Review by EllipticEllipsis — COMMENTED** [review]
One minor thing. It's not a pretty match, but at least it works... did you check every possible one of the 32 possible temp combinations?

**Comment by hensldm** [comment]
Yeah, only two combinations work, this one or one with `OPEN` and `func_8012C28C` having the temp and `CLOSE` not using the temp.

**Review by EllipticEllipsis — APPROVED** [review]
Not unheard of to have them use the temp at the top but not the bottom, but no better really. Nice work, being so thorough!

**Review by engineer124 — APPROVED** [review]

**Review by AngheloAlf — APPROVED** [review]

**Review by rylieb — APPROVED** [review]

**Review by Kenix3 — APPROVED** [review]

**Pull request merged**